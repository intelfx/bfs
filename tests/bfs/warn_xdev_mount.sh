# Regression test: don't crash if -mount is the last option
bfs_diff basic -warn -xdev -mount
