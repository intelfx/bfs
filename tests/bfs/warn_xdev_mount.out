basic
basic/a
basic/b
basic/c
basic/c/d
basic/e
basic/e/f
basic/g
basic/g/h
basic/i
basic/j
basic/j/foo
basic/k
basic/k/foo
basic/k/foo/bar
basic/l
basic/l/foo
basic/l/foo/bar
basic/l/foo/bar/baz
